docs(cloud-security): correct runtime package tracking version + remove WP cost warning (#36954)
DataDog/documentation: content/en/security/cloud_security_management/setup/agent/docker.md

These signals power vulnerability prioritization in Cloud Security, surfacing findings where vulnerable code is confirmed running in production.

**Requirements**:
- Datadog Agent **7.79.0 or later**
- Linux only (eBPF dependency)

> **Note**: Use Datadog Agent **7.79.0 or later**. Earlier Agent versions enable this feature through Workload Protection, which can affect Workload Protection usage. From 7.79.0, runtime package prioritization runs independently and does not affect Workload Protection usage.

Reviewer: ```suggestion
**Note**: Use Datadog Agent **7.79.0 or later**. Earlier Agent versions enable this feature through [Workload Protection][4] and can affect its usage. From 7.79.0, runtime package prioritization runs independently and does not affect its usage.
```
I noticed that other instances of this note linked out to WP, so added the link here as well; feel free to remove if you think it's too much

  registry.datadoghq.com/agent:7
{{< /code-block >}}

> **Note**: `DD_SBOM_ENRICHMENT_USAGE_ENABLED=true` is in Preview and requires Datadog Agent **7.79.0 or later**. From 7.79.0, runtime package prioritization runs independently of [Workload Protection][4] and does not affect Workload Protection usage. See the [Runtime Package Prioritization](#runtime-package-prioritization-preview) section for more details.

Reviewer: ```suggestion
**Note**: `DD_SBOM_ENRICHMENT_USAGE_ENABLED=true` is in Preview and requires Datadog Agent **7.79.0 or later**. From 7.79.0, runtime package prioritization runs independently of [Workload Protection][4] and does not affect its usage.
```
This content is already in that section, so I don't think we need to link to itself
